src/input-helper.ts

```diff
@@ -272,13 +275,24 @@ async function getCommitMessagesFromPullRequest(
   interface EdgeItem {
     node: {
       commit: {
-        message: string
+        message: string,
+        parents: {
+          totalCount: number
+        }
       }
     }
   }
 
   if (repository.pullRequest) {
-    messages = repository.pullRequest.commits.edges.map(function(
+    messages = repository.pullRequest.commits.edges.filter(function (
+      edge: EdgeItem
+    ): boolean {
+      if (edge.node.commit.parents.totalCount > 1) {
+        // Skip merge commits (which have more than 1 parent commit)
+        return false;
+      }
+      return true;
+    }).map(function (
```

Comment:
Here we perform the filtering locally, skipping merge commits. End result, we will just pretend those commits don't exist.